cli/src/main/groovy/org/sonatype/nexus/cli/Repository2.groovy

```diff
@@ -0,0 +1,33 @@
+package org.sonatype.nexus.cli
```

Comment:
`RepositoryManagerV2.groovy` may make more sense... when first reading I thought you were just implementing the same thing twice for debugging or testing or whatnot.